projects/wave-core/src/lib/operators/dialogs/source-operator-list/source-operator-list.component.ts

```diff
@@ -42,6 +43,7 @@ export class SourceOperatorListComponent implements OnInit {
     DrawFeaturesComponent = OlDrawFeaturesComponent;
     PolygonSelectionComponent = CountryPolygonSelectionComponent;
 
+    ChronicleDbSourceComponent = ChronicleDbSourceComponent;
```

Comment:
we also should omit changes to this file

Reply:
merged with the other PR / master